### PR TITLE
fix: run real fedimint-ln-gateway tests

### PR DESCRIPTION
**Pull request by bradleystachurski**
🙈

### CODE REVIEW

**Review by dpc — DISMISSED** [review]

File: scripts/tests/backend-test.sh
Comment (justinmoon) [inline scripts/tests/backend-test.sh:47]: So was this a no-op?

**Review by justinmoon — DISMISSED** [review]

File: scripts/tests/backend-test.sh
Comment (bradleystachurski) [reply scripts/tests/backend-test.sh:47]: Correct 👍

**Comment by bradleystachurski** [comment]
Hmm, odd this passed the PR CI but failed the merge queue

```
104719   │ 2024-02-22T14:07:40.2486496Z 00:05:48      Summary [  72.254s] 8/20 tests run: 7 passed, 1 failed, 0 skipped
104720   │ 2024-02-22T14:07:40.2486964Z 00:05:48         FAIL [  33.834s] fedimint-ln-gateway::gatewayd-integration-tests test_gateway_can_pay_ldk_node
```

More details

```
104635   │ 2024-02-22T14:07:40.2444588Z 00:05:21 thread 'test_gateway_can_pay_ldk_node' panicked at gateway/ln-gateway/tests/integration_tests.rs:187:13:
104636   │ 2024-02-22T14:07:40.2447898Z 00:05:21 assertion failed: `Canceled { error: OutgoingPaymentError { error_type: LightningPayError { lightning_error: FailedPayment { fail
        │ ure_reason: "FailureReasonNoRoute" } }, contract_id: bec4bb4907c5469edc8453946401ff15c6d5c80f3429967cb0662cb75dd24644, contract: Some(OutgoingContractAccount { amount:
        │  Amount { msats: 250000 }, contract: OutgoingContract { hash: e569bd88a54133c2b6901afaced1ebc4377e6969a11fff0fa93a80974f474363, gateway_key: PublicKey(18fde48e4acba6f4
        │ 0e1e6c70eaefd04c8f716ee50117968b81c5a91797ea875c1b3354bc00f8739085e4b4a8a9628dfa7b5eefd77ef063e4de0e8d32fb34870c), timelock: 758, user_key: PublicKey(25322123f8cdef54e
        │ 64ed7b5e6e7d438df45fdf4077284b0b0dd5d9661e7c51a267409a4fdd48931956c69cc9a705d84c305c4f593cbaf4ee3db3101501195bf), cancelled: false } }) } }` does not match `GatewayExt
        │ PayStates::Preimage { .. }`
```

**Review by elsirion — APPROVED** [review]
Fiat CI strikes again :see_no_evil:

**Comment by bradleystachurski** [comment]
@elsirion Would be okay if we ignore `test_gateway_can_pay_ldk_node` so we don't have a bunch of flakes on master? We can remove the `#[ignore]` in https://github.com/fedimint/fedimint/pull/4354 before merging.

**Review by m1sterc001guy — APPROVED** [review]

**Comment by m1sterc001guy** [comment]
I can help debug the flaky `ldk-node` test

**Review by elsirion — APPROVED** [review]

**Comment by bradleystachurski** [comment]
> removed this pull request from the merge queue due to no response for status checks

We may need to increase the status check timeout: https://github.com/orgs/community/discussions/47293

Can someone with permissions try increasing?

**Comment by bradleystachurski** [comment]
> Can someone with permissions try increasing?

Actually let's hold off increasing. Seeing other PRs fail with the same no response error, so might be a clogged merge queue.

**Review by dpc — APPROVED** [review]

**Pull request merged**
